### DISCUSSION TITLE
Mac下nvim侧边栏不显示图标

### DISCUSSION THREAD

**Discussion by UniStart2** [nshen/learn-neovim-lua#152] [Q&A] [answered]
我按照步骤走的，但是弄完之后发现侧边栏的文件不显示图标，有大佬能指点一下么。我的插件都是以ssh形式下载的，都是安装好的，PackerSync都是OK的。

问题如图下图所示：
[![zyzJ2V.png](https://s1.ax1x.com/2022/12/05/zyzJ2V.png)](https://imgse.com/i/zyzJ2V)
plugin-config/nvim.tree.lua配置如下：
![Vgqpy.png](https://i.328888.xyz/2022/12/05/Vgqpy.png)

**Comment by nshen** [comment, marked as answer]
图标字体要装 nerdfont ，然后终端设置字体，你这个终端没见过，不知道支持不，mac下我只 用过 iterm2 或 alacritty。

**Reply by UniStart2** [reply]
终端用的是wrap，我确实是忘记设置终端的字体为Nerd Font字体了，问题解决了，谢谢
